package.json

```diff
@@ -11,7 +11,7 @@
   ],
   "scripts": {
     "build": "fedx-scripts webpack",
-    "i18n_extract": "fedx-scripts formatjs extract",
+    "i18n_extract": "formatjs extract --format node_modules/@openedx/frontend-build/lib/formatter.js --ignore {src,plugins}/**/*.json --out-file ./temp/babel-plugin-formatjs/Default.messages.json -- {src,plugins}/**/*.js*",
```

Comment:
Thanks @igobranco for the pull request and really sorry for the delay. Somehow I didn't include this pull request on my todo list.

I'll test locally, but one question. I noticed that you're setting the file to become `./temp/babel-plugin-formatjs/Default.messages.json`, what's reason for this specific file choice and why did you mention it explicitly?

Are you aware of the specifics of `fedx-scripts formatjs extract`? What features do we expect to lose when removing the `fedx-scripts` meta-script?